vocabularies/Org.OData.Aggregation.V1.xml

```diff
@@ -205,84 +205,195 @@
+        </Annotation>
+      </TypeDefinition>
+
+      <TypeDefinition Name="HierarchyNode" UnderlyingType="Edm.String">
+        <Annotation Term="Core.Description" String="Property path identifying the node in a recursive hierarchy" />
+        <Annotation Term="Core.LongDescription">
+          <String>Every recursive hierarchy function defined in this vocabulary has an overload with a parameter of this type.
+          This parameter specifies the property of the `Entity` (the function's binding parameter) whose primitive value identifies the entity
+          as a node in the recursive hierarchy that is evaluated by the function.
+          Which hierarchy this is depends on the other parameters `HierarchyNodes` and [`Hierarchy`](#HierarchyQualifier).
+          The entity given in the binding parameter need not belong to `HierarchyNodes`, only its node value is compared
+          to the node values in the recursive hierarchy.
+          The function overload then disregards a [`RecursiveHierarchy`](#RecursiveHierarchy) annotation on the entity if there is one.</String>
+        </Annotation>
+      </TypeDefinition>
+
       <Function Name="isroot" IsBound="true">
-        <Annotation Term="Core.Description"
-          String="Returns true for a given entity, if and only if the value of the node property of the specified hierarchy annotation is the root of the hierarchy in the given entity set of hierarchy nodes" />
-        <Parameter Name="Entity" Type="Edm.EntityType" Nullable="false" />
-        <Parameter Name="Nodes" Type="Edm.String" Nullable="false">
-          <Annotation Term="Core.Description" String="Name of the entity set holding the nodes of the recursive hierarchy" />
-        </Parameter>
-        <Parameter Name="Hierarchy" Type="Edm.String" Nullable="false">
-          <Annotation Term="Core.Description" String="Qualifier to identify the `RecursiveHierarchy` annotation applied to the type of the entity set referenced in `Nodes`" />
-        </Parameter>
-        <ReturnType Type="Edm.Boolean" Nullable="false" />
+        <Annotation Term="Core.Description" String="Is the entity the root of the specified hierarchy?" />
+        <Parameter Name="Entity" Type="Edm.EntityType" Nullable="true" />
+        <Parameter Name="Hierarchy" Type="Aggregation.HierarchyQualifier" Nullable="false" />
+        <ReturnType Type="Edm.Boolean" Nullable="true" />
+      </Function>
+      <Function Name="isroot" IsBound="true">
+        <Annotation Term="Core.Description" String="Is the entity the root of the specified hierarchy?" />
+        <Parameter Name="Entity" Type="Edm.EntityType" Nullable="true" />
+        <Parameter Name="NodePropertyPath" Type="Aggregation.HierarchyNode" Nullable="false" />
+        <Parameter Name="HierarchyNodes" Type="Collection(Edm.EntityType)" Nullable="false" />
+        <Parameter Name="Hierarchy" Type="Aggregation.HierarchyQualifier" Nullable="false" />

```

Comment:
I'd move this to directly after `Entity`, just for readability and similarity to the other overload.

Also applies to other functions and overloads: `Entity` first, `Hierarchy` second,